packages/cluster-ui/src/util/appStats/appStats.ts

```diff
@@ -107,6 +107,17 @@ export function addExecStats(a: ExecStats, b: ExecStats): Required<ExecStats> {
   };
 }
 
+function combinesUnique<T>(a: Array<T>, b: Array<T>): Array<T> {
```

Comment:
I like to use `Set` to create unique arrays. You may consider doing something like,

```js
const combineUnique = <T>(a: Array<T>, b: Array<T>): Array<T> => [...new Set([...a, ...b])];
```

This could also be added to `src/utils`.

.... On second thought, I think I would rather have a `unique` utility function,

```js
const uniq = <T, >(a: T[]): T[] => [...new Set([...a])];
```

And then use this function to return a unique concatenation of  multiple arrays. Like so,

`nodes: unique([...a.nodes, ...b.nodes]),`